### PR TITLE
Jorge hash

### PR DESCRIPTION
**Pull request by Georgeg10499**
This commit gives the user the ability to navigate to the hashtag pages, but I still need to figure out how each message and conversation have their own hashtag identifier. Also, please disregard the commented statements in the for loops this will be implemented after we figure out how to add #ids

### CODE REVIEW

File: src/main/java/codeu/controller/HashListServlet.java [hunk 1]
```diff
@@ -43,17 +45,17 @@
   @Override
   public void init() throws ServletException {
     super.init();
-   // setHashtagStore(UserStore.getInstance());
+   setHashtagStore(HashtagStore.getInstance());
```
Comment (chilimalone) [inline src/main/java/codeu/controller/HashListServlet.java]: nit: +1 indent

File: src/main/java/codeu/controller/HashListServlet.java [hunk 2]
```diff
@@ -62,12 +64,12 @@ public void init() throws ServletException {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    List<String> hashtags = new ArrayList<>();
-    hashtags.add("happy");
-    hashtags.add("funny");
-    hashtags.add("food");
-    hashtags.add("favorite places");
-    hashtags.add("family");
+    List<Hashtag> hashtags = new ArrayList<>();/*hashtagStore.getAllHashtags();*/
```
Comment (chilimalone) [inline src/main/java/codeu/controller/HashListServlet.java]: let's remove the commented out code if it's no longer needed

File: src/main/java/codeu/controller/HashtagServlet.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    List<Conversation> conversations = conversationStore.getAllConversations();
+    List<Message> messages = messageStore.getMessages();
+
+    String requestUrl = request.getRequestURI();
+    String hashtagTitle = requestUrl.substring("/hashtag/".length());
+    Hashtag hashtag = hashtagStore.getHashtagWithHashTitle(hashtagTitle);
+
+    for(Conversation c: conversations){
+      /*if(hashtag.getHashTitle().equals("c.getHashtag()")) {*/
```
Comment (chilimalone) [inline src/main/java/codeu/controller/HashtagServlet.java:87]: ditto remove commented out code if they're no longer necessary

**Review by chilimalone — APPROVED** [review]
Looks good! just a few small nits

**Review by jayyduran0108 — APPROVED** [review]

**Pull request merged**